src/main/java/net/dv8tion/jda/internal/handle/MessageReactionHandler.java

```diff
@@ -147,11 +147,7 @@ protected Long handleInternally(DataObject content)
             );
         }
 
-        // reaction remove has null name sometimes
-        if (emoji.isNull("name"))
-            emoji.put("name", "");
         EmojiUnion rEmoji = Emoji.fromData(emoji);
-
```

Comment:
For consistency reason I would add that newline back or remove it too from the `MessageReactionClearEmojiHandler`.

Reply:
I don't think anything mentioned matters at all. in `MessageReactionClearEmojiHandler`, I think the variables passed into `new MessageReaction(...)` were just meant to be separated, same with `MessageReactionHandler` and the check.